igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildCache.groovy

```diff
@@ -0,0 +1,83 @@
+package com.netflix.spinnaker.igor.build
+
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import redis.clients.jedis.Jedis
+import redis.clients.jedis.JedisPool
+
+
+/**
+ * Shared cache of build details
+ */
+@SuppressWarnings(['PropertyName', 'DuplicateNumberLiteral'])
+@Service
+@Slf4j
+class BuildCache {
```

Comment:
Im not sure if this is what you want, but I did this https://github.com/gardleopard/igor/commit/c68dbabf7d9a4b5d50eae3b54d67f1f698a54e63